v1/zl/ib/zlib

```diff
@@ -1,14 +1,24 @@
 {
   "category": "compression",
-  "contact": "dylan-lang@googlegroups.com",
-  "description": "Simple Dylan zlib wrapper",
-  "keywords": [],
+  "contact": "dylan-lang@google.com",
```

Comment:
"@googlegroups.com" was correct.  "@google.com" would be an internal-to-Google mailing list.

Reply:
Fixed.